todo_react_1/src/ListItem.js

```diff
@@ -0,0 +1,37 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+function ListItem(props) {
+    const labelStyles = ['todolist__label']
+
+    if (props.item.done) {
+        labelStyles.push('decoration-done')
+    }
+
+    return (
+        <li>
+            <form>
+                <input className="todolist__checkbox" type="checkbox" id={props.item.id}
+                       onChange={() => props.onChangeDone(props.item.id)}/>
+                <label className={labelStyles.join(' ')} htmlFor={props.item.id}>
```

Comment:
Please use `classnames` to rewrite this on smth like `classnames('todolist__label',  props.item.done && 'decoration-done')`